msal/tests/Test.MSAL.NET.Unit.net45/RequestsTests/InteractiveRequestTests.cs

```diff
@@ -52,28 +52,18 @@ namespace Test.MSAL.NET.Unit.RequestsTests
     public class InteractiveRequestTests
     {
         private readonly MyReceiver _myReceiver = new MyReceiver();
-        private TokenCache _cache;
-        private ITelemetryManager _telemetryManager;
-        private IValidatedAuthoritiesCache _validatedAuthoritiesCache;
 
         [TestInitialize]
         public void TestInitialize()
         {
             TestCommon.ResetStateAndInitMsal();
-            _validatedAuthoritiesCache = new ValidatedAuthoritiesCache();
-            _telemetryManager = new TelemetryManager(_myReceiver);
-
-            _cache = new TokenCache
-            {
-                TelemetryManager = _telemetryManager,
-            };
         }
 
         [TestCleanup]
         public void TestCleanup()
         {
-            _cache.TokenCacheAccessor.ClearAccessTokens();
-            _cache.TokenCacheAccessor.ClearRefreshTokens();
+            //_cache.TokenCacheAccessor.ClearAccessTokens();
+            //_cache.TokenCacheAccessor.ClearRefreshTokens();
```

Comment:
what's going to happen here?

Reply:
I suppose the cache has been removed, thus the clearnup can be removed as well?

---
In reply to: [236820555](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1434#discussion_r236820555) [](ancestors = 236820555)

Reply:
Correct, the cache isn't a member so we'll create a new one.  i think these were unnecessary even previously since we were recreating the _cache variable during TestInitialize.  I've removed the dead code now.